sechub-schedule/src/main/java/com/daimler/sechub/domain/schedule/strategy/SchedulerStrategyId.java

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.domain.schedule.strategy;
+
+public enum SchedulerStrategyId {
+    FirstComeFirstServe("first-come-first-serve"),
+    OnlyOneScanPerProjectAtATime("only-one-scan-per-project-at-a-time");
+
+    private String strategy;
+
+    SchedulerStrategyId(String strategyString) {
+
+        if (strategyString == null || strategyString.isEmpty()) {
+            throw new IllegalArgumentException("strategyString may not be null!");
```

Comment:
maybe two different if statements with two different illegal argument exceptins 
1. may not be null 
2. may not be empty?

Reply:
done